### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Dadadon**
- Set up Postgres database.
- Built your project schema.
- Created and ran the necessary migration files.
- Table and column names match the ERD diagram.
- Photo for users table is a link to an image
- Foreign keys are included.
- All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
```
Comment (rotimiazeez) [inline README.md]: - Please keep in mind that the logo provided in your README file is broken. Kindly fix it 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Dadadon 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
- It will be nice if you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the #update_post_counter method every time a new post has been saved to database, then make the update_post_counter method private as seen below:
```
  # app/models/post.rb
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```

- It will be nice if you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the #update_like_counter method every time a new like has been saved to database, then make the update_like_counter method private as seen below:
```
  # app/models/like.rb
  after_save :update_like_counter

  private

  def update_like_counter
    author.increment!(:likes_counter)
  end
```

- It will be nice if you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the #update_comment_counter method every time a new comment has been saved to database, then make the update_comment_counter method private as seen below:
```
  # app/models/comment.rb
  after_save :update_comment_counter

  private

  def update_comment_counter
    author.increment!(:comments_counter)
  end
```
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: README.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
 <a name="readme-top"></a>
 <div align="center">
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Island Blogs</b></h3>
+  <h3><b>Island Blog</b></h3>
```
Comment (codecaiine) [inline README.md:5]: - [ ] [OPTIONAL] I would like to strongly recommend you customize your license link by adding your name and your project title in order to make it more professional 👍

**Review by codecaiine — APPROVED** [review]
Hi @Dadadon , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great request changes application ✔️ 
- Great suggestion changes implementation ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**